woodwork/utils.py

```diff
@@ -23,6 +23,41 @@ def import_or_none(library):
         return None
 
 
+def import_or_raise(library, error_msg):
+    '''
+    Attempts to import the requested library.  If the import fails, raises an
+    ImportError with the supplied error message.
+
+    Args:
+        library (str): the name of the library
+        error_msg (str): error message to return if the import fails
+    '''
+    try:
+        return importlib.import_module(library)
+    except ImportError:
+        raise ImportError(error_msg)
+
+
+def import_and_configure_koalas(compute_ops_on_diff_frames=True):
```

Comment:
It'd be cool if we could make this a generic `import_and configure_package(package, **args)`. Doesn't have to be in this PR, but maybe something to think about in the future.

Reply:
Totally agree! I think that work should happen if/when we add more "global config" options like this, at which point we should probably add a config object to encapsulate that stuff.